app/views/questions/edit.html.erb

```diff
@@ -0,0 +1,50 @@
+<h2>Question</h2>
+
+<div class="row">
+  <div class="col s12 m12">
+    <div class="card-panel white">
+      <span class="black-text">
+          <%= form_for @question do |f| %>
+        <div id="question-title" >
```

Comment:
Not sure if we need all this html on the edit page. Is this what you were talking about in your initial PR comment about styling? If so we can take a look at it tomorrow when we get together and break it down.